src/components/Tree/TreeTitle.tsx

```diff
@@ -0,0 +1,79 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import { UIComponent, childrenExist, customPropTypes, createShorthandFactory } from '../../lib'
+import { ComponentSlotStyle, ComponentVariablesInput } from '../../themes/types'
+import { treeTitleBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+
+export type TreeTitleProps = {
+  as?: any
+  children?: React.ReactChildren
+  content?: React.ReactNode
+  styles?: ComponentSlotStyle
+  variables?: ComponentVariablesInput
+  active?: boolean
+  hasSubtree?: boolean
+}
+
+class TreeTitle extends UIComponent<TreeTitleProps, any> {
+  static create: Function
+
+  static className = 'tree-title'
```

Comment:
As I remember, we're following BEM for `className`

```suggestion
  static className = 'ui-tree__title'
```